runner.go

```diff
@@ -102,20 +129,69 @@ func generateUserSeed() error {
 	}
 }
 
+func appendFile(dst io.Writer, path string) error {
+    if src, err := os.OpenFile(path, os.O_RDONLY, 0644); err != nil {
+        log.Print("Failed to open file: ", err)
+        return err
+    } else {
+        defer src.Close()
+        if _, err := io.Copy(dst, src); err != nil {
+            log.Print("Failed to read file: ", err)
+            return err
+        }
+    }
+    _, err := dst.Write([]byte("\n"))
+    return err
+}
+
 func enableSplunkAPI() error {
+
 	if serverFile, err := os.OpenFile(os.ExpandEnv(ServerConfigPath), os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0644); err != nil {
 		return err
 	} else {
 		defer serverFile.Close()
-		_, err = fmt.Fprintf(serverFile, `[sslConfig]
-enableSplunkdSSL = false
-[httpServer]
-acceptFrom = 127.0.0.1/8
-[proxyConfig]
-http_proxy = %s
-https_proxy = %s
-no_proxy = %s
-`, os.Getenv("HTTP_PROXY"), os.Getenv("HTTPS_PROXY"), os.Getenv("no_proxy"))
+
+        if err := exec.Command("/usr/bin/update-ca-trust").Run(); err != nil {
```

Comment:
strange indent.  Should this be within the "else"?

Reply:
Looking at what was added to this function, likely just need to update the indent  of added lines to be tabs instead of four spaces to match the file's style.  I'm guessing this is just from IDE setting diffs.